pom.xml

```diff
@@ -54,71 +54,6 @@
         <url>https://github.com/apache/incubator-kyuubi/issues</url>
     </issueManagement>
 
-    <developers>
-        <developer>
-            <id>aajisaka</id>
-            <name>Akira Ajisaka</name>
-            <email>aajisaka@apache.org</email>
-            <organization>Yahoo</organization>
```

Comment:
@pan3793 
My organization is "Yahoo Japan Corporation". Would you include this fix in 1.3.0?
Sorry, I forgot to check it.

Reply:
Apologize for this mistake, I'm going to cancel the current RC7 vote and raise a new one.

Reply:
Thank you @pan3793!